README.md

```diff
@@ -298,6 +298,7 @@ Please do **not** create a public GitHub issue.
 - [Meeting readiness checker](#meeting-readiness-checker)
 - [Selecting an Audio Profile](#selecting-an-audio-profile)
 - [Starting a Messaging Session](#starting-a-messaging-session)
+- [Amazon Voice Focus and Echo Reduction](#amazon-voice-focus-and-echo-reduction)
```

Comment:
I do not think this is the right place to put this section. All the other items in H2 are usage examples with code. While this new section seems to be just a feature description.